src/backend/distributed/commands/alter_table.c

```diff
@@ -487,6 +488,9 @@ AlterTableSetAccessMethod(TableConversionParameters *params)
 TableConversionReturn *
 ConvertTable(TableConversionState *con)
 {
+	bool oldEnableLocalReferenceForeignKeys = EnableLocalReferenceForeignKeys;
+	SetLocalEnableLocalReferenceForeignKeys(false);
```

Comment:
Could use a comment on why we need to do this.

Reply:
1ec3c304956b28fb39ffe63a72abc413f4383e72